### DISCUSSION TITLE
highlight linenos does not work on distill post?

### DISCUSSION THREAD

**Discussion by gladscortes** [alshedivat/al-folio#629] [Q&A]
Hi, I'm new to github pages and jekyll

On the sample DISTILL post (/blog/2021/distill/), I changed the code FROM

{% highlight javascript %}
var x = 25;
function(x) {
  return x * x;
}
{% endhighlight %}

TO

{% highlight javascript linenos %}
var x = 25;
function(x) {
  return x * x;
}
{% endhighlight %}

AND the rendered result was a code block that looks like
+------------------------------------------------------+

1
2
3
4

var x = 25;
function(x) {
  return x * x;
}
+------------------------------------------------------+
ON both Brave and Safari browsers (using macOS 12.3.1)

ALSO, syntax highlighting doesn't seem to work for R on the distill post:
all the code was rendered purple.
meanwhile, syntax highlighting works fine on the "post" layout

any suggestions? for
1. how to put line numbers successfully on distill, and
2. how to enable syntax highlighting for R on distill

Thanks

**Comment by riasat-sheikh** [comment]
Same thing is happening for me as well. Also, fortran highlight isn't working in distill mode but works on post mode.
